src/get-user-code-frame.js

```diff
@@ -0,0 +1,70 @@
+// We try to load node dependencies
+let chalk = null
+let readFileSync = null
+let codeFrameColumns = null
+
+try {
+  const nodeRequire = module && module.require
+
+  readFileSync = nodeRequire.call(module, 'fs').readFileSync
+  codeFrameColumns = nodeRequire.call(module, '@babel/code-frame')
+    .codeFrameColumns
+  chalk = nodeRequire.call(module, 'chalk')
+} catch {
+  // We're in a browser environment
+  /* istanbul ignore next */
+  console.warn(
+    'Printing the user trace is not supported in a browser environment',
+  )
+}
```

Comment:
Let's not bother warning. This is a nice-to-have and folks wouldn't expect this feature anyway.

```suggestion
} catch {
  // We're in a browser environment
}
```

Reply:
This one as well